problem1Spec.js

```diff
@@ -0,0 +1,41 @@
+describe("Problem1 test 1", function() {
+	var objectProblem1;
+	var testArray;
+	
+	beforeEach(function() {
+		objectProblem1 = new ProblemsSolved();
+	});
+  
+  it("Sum of even numbers in case of exist even number/numbers", function() {
+	testArray = [4,4,4,4,5];
+	expect(objectProblem1.SumOfEvenNumberInArray(testArray)).toEqual(16);
+  });
+});
+
+describe("Problem1 test 2", function() {
+	var objectProblem1;
+	var testArray;
+	
+	beforeEach(function() {
+		objectProblem1 = new ProblemsSolved();
+	});
+  
+  it("Sum of even numbers in case of not exist even number/numbers", function() {
+	testArray = [1,3,1,1,5];
+	expect(objectProblem1.SumOfEvenNumberInArray(testArray)).toEqual(0);
+  });
+});
+
+describe("Problem1 test 3", function() {
+	var objectProblem1;
+	var testArray;
+	
+	beforeEach(function() {
+		objectProblem1 = new ProblemsSolved();
+	});
+  
+  it("Sum of even numbers in case of size of array is 0", function() {
+	testArray = [];
+	expect(objectProblem1.SumOfEvenNumberInArray(testArray)).toEqual(0);
+  });
+});
```

Comment:
I do not know how to work professionally, but maybe it's much easier if each test should be in a separate file.